python/cugraph/ktruss/ktruss_subgraph.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import cugraph
+from cugraph.structure.graph import Graph
+
+
+from cugraph.ktruss import ktruss_subgraph_wrapper
+
+
+def ktruss_subgraph(G,k):
```

Comment:
Is this called `k_truss` in Nx?

Reply:
Correct. 
https://networkx.github.io/documentation/stable/reference/algorithms/generated/networkx.algorithms.core.k_truss.html

Nx does not have a maximal k-truss algorithm.  So we have two functions instead of one.

Reply:
Or we will have two functions once the second one is completed and exposed.